lib/bulk/common.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-const BSON = require('../utils').retrieveBSON();
-const { Long, ObjectID } = BSON;
+const { BSON } = require('../deps');
```

Comment:
does `const { BSON: { ObjectId, Long } } = require('../deps');` not work here?

Reply:
I totally can do this! I was just thinking as I was hunting around for usages of BSON wouldn't it be easier if someone in the future had to do this if we namespaced our usage, and it only makes sense cus its an external lib, if importing something from within the driver dropping the namespace would always be the preference. It's no trouble to revert, lmk!

Reply:
I have a mild preference for importing the types directly. It seems to me that now that we can use features like object destructuring we should try to make similar changes everywhere, and it would be strange to have a different rule just for these "dynamically imported" modules.